capellambse/aird/parser/_filters/global.py

```diff
@@ -12,6 +12,40 @@
 from . import FilterArguments, composite, global_filter
 
 XT_CEX_FEX_ALLOCATION = "org.polarsys.capella.core.data.fa:ComponentExchangeFunctionalExchangeAllocation"
+EXCHANGES_WITH_ROLES = frozenset(
+    ("Association", "Aggregation", "Composition", "Generalization")
+)
+
+
+@global_filter("hide.association.labels.filter")
+def hide_association_labels(
+    args: FilterArguments, flt: lxml.etree._Element
+) -> None:
+    """Hide names of roles on an exchange."""
+    del flt
+
+    classes = EXCHANGES_WITH_ROLES
+    for obj in args.target_diagram:
+        if not (isinstance(obj, aird.Edge) or obj.styleclass in classes):
+            continue
+
+        assert isinstance(obj, aird.Edge)
```

Comment:
This would look more elegant (and wouldn't need a double-check in the form of an assertion) if we simply ignored everything that's not an Edge, i.e.:

```suggestion
        if not isinstance(obj, aird.Edge) or obj.styleclass not in classes:
            continue
```